### PR TITLE
PCI-482 Basic Spring Boot application

### PR DESCRIPTION
**Pull request by VaughanJackson123**

### CODE REVIEW

File: mvnw [hunk 1]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (Mathsh) [inline mvnw]: is this mvnw file needed?

**Comment by Mathsh** [comment]
usually in a start up project we include the .gitignore file, a Makefile, start.sh and readme.

File: mvnw (hunk 1 above)
Comment (VaughanJackson123) [reply mvnw]: No, I think you're right. We don't really need this maven wrapper given that we have maven and we will be using a `start.sh` script to run the application. I'll remove it.

**Comment by VaughanJackson123** [comment]
Yes, @Mathsh, thanks for this suggestion. I will look into adding these files at this stage too, rather than later on as part of the Vagrant routing.

**Pull request merged**